lib/Feature/Feature.cpp

```diff
@@ -1,57 +1,34 @@
 #include "vara/Feature/Feature.h"
 
+#include <experimental/iterator>
```

Comment:
```suggestion
#include <iterator>
```
Why would we need experimental stuff here?